### PR TITLE
Central system design polish

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Adds a variety of small tweaks to VxCentralSystem and VxAdmin to improve how they use the desktop theme.

## Demo Video or Screenshot
Central Scan - adjudication
Screenshot 2023-11-22 at 11 24 31 AM

Admin - Election Manager - Election screen (formerly Ballots screen)
Screenshot 2023-11-22 at 1 04 58 PM

Admin - Election Manager - tallies and marking as official
https://github.com/votingworks/vxsuite/assets/530106/54dd0dbd-f0c6-4028-9bc2-9bbc808b0148

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: apps/admin/frontend/src/components/app_routes.tsx
Comment (adghayes) [inline apps/admin/frontend/src/components/app_routes.tsx:1]: someone finally renamed this file!

File: apps/admin/frontend/src/screens/election_screen.tsx [hunk 1]
```diff
@@ -46,14 +53,27 @@ export function DefinitionScreen(): JSX.Element {
...
+          <React.Fragment>
+            <P>
+              Save the Ballot Package to USB to configure VxCentralScan or
+              VxScan.
+            </P>
+            <P>
+              <ExportElectionBallotPackageModalButton />
+            </P>
+          </React.Fragment>
+        )}
```
Comment (adghayes) [inline apps/admin/frontend/src/screens/election_screen.tsx:76]: In development, I've always wanted to export the ballot package as the system admin 🤷 , but making a decision on that doesn't need to happen in this PR / code freeze.

File: apps/admin/frontend/src/components/app_routes.tsx [hunk 2]
```diff
@@ -145,8 +144,8 @@ export function ElectionManager(): JSX.Element {
   // Election manager UI
   return (
     <Switch>
-      <Route exact path={routerPaths.ballotsList}>
-        <BallotListScreen />
```
Comment (adghayes) [inline apps/admin/frontend/src/components/app_routes.tsx:149]: Is this a file that could be deleted now?

**Review by adghayes — APPROVED** [review]
LGTM! I like pretty much all the changes, and agree with @mattroe's feedback provided over Slack. I do like the arrangement of the buttons above the table. And the mark official flow is better than its ever been.

I don't have a view of the central-scan test banner without getting on your branch, but assuming that it's fine. The admin test banner on the Tally page is the only thing that I wanted to play with a bit:

Screen Shot 2023-11-22 at 4 38 18 PM

I am fine with shipping it as-is, but I would be curious to see it with the test mode banner above the CVRs header rather than below it. It feels like the "Load CVRs" and "Remove CVRs" button belong below the header and are being split by the banner.

File: apps/admin/frontend/src/components/app_routes.tsx (hunk 2 above)
Comment (adghayes) [reply apps/admin/frontend/src/components/app_routes.tsx:149]: The file containing `BallotsListScreen`

Comment (jonahkagan) [reply apps/admin/frontend/src/components/app_routes.tsx:149]: Yup, meant to do that, thanks

File: apps/admin/frontend/src/screens/election_screen.tsx (hunk 1 above)
Comment (jonahkagan) [reply apps/admin/frontend/src/screens/election_screen.tsx:76]: Oh interesting. I agree we can discuss outside of this PR

**Comment by jonahkagan** [comment]
> I don't have a view of the central-scan test banner without getting on your branch, but assuming that it's fine.

Yeah, no visual change to that - just changed the implementation to take advantage of `Card.color`.

**Comment by jonahkagan** [comment]
> I would be curious to see it with the test mode banner above the CVRs header rather than below it.

I agree it looks a bit nicer this way. I'm down to change it.
Screenshot 2023-11-22 at 3 03 25 PM

We should also consider how it looks after marking results as official (though I'm not sure if anyone will do that during L&A)

Screenshot 2023-11-22 at 3 03 01 PM

**Comment by adghayes** [comment]
The test mode + mark official state is one that we have to support (I was told when editing report titles), for a complete parallel between normal and test mode, but we don't have to worry about how it looks since we believe no one uses it and, if they do, they know it's just for testing. My instinct is to have the test banner always come first, like in these two cards, but given rarity of this screen I don't think it really matters.

**Pull request merged**